Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -190,10 +195,14 @@ function initialize(model) {
     .then(function (loader) {
       var components = loader.components;
       var content = model._content;
-      var featureMetadata = components.featureMetadata;
+
+      // For 3D Tiles 1.0 formats, the feature metadata is owned by the Cesium3DTileContent classes.
+      var featureMetadata = defined(content.featureMetadata)
```

Comment:
Similar to @ptrgags's comment below, make sure to check that `content` is defined first. This will break code that create a `ModelExperimental` by itself like the `buildings-metadata` sandcastle in https://github.com/CesiumGS/cesium/issues/9832. Thankfully this is caught by unit tests but only when run locally, so make sure to always do that before opening a PR.

Reply:
Added the required checks.